Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1133,9 +1159,9 @@ theorem le_sup {ι : Type u} (f : ι → Ordinal.{max u v}) : ∀ i, f i ≤ sup
   Ordinal.le_iSup f i
 
 /-- `ciSup_le_iff'` whenever the outputs live in a higher universe than the inputs. -/
-protected theorem iSup_le_iff {ι : Type u} {f : ι → Ordinal.{max u v}} {a} :
+protected theorem iSup_le_iff {ι} {f : ι → Ordinal.{u}} {a : Ordinal.{u}} [Small.{u} ι] :
```

Comment:
Can't you generalize `le_iSup`, `iSup_le_iff`, and `iSup_le` in the exact same way? That would ensure consistency across these four functions.

Reply:
`le_iSup` became a bit worse at filling in the function argument which is unfortunate, and besides that there are a bunch of similarly generalizable lemmas in this file that I'll deal with in a seperate PR (tomorrow if I have time).

Reply:
#16996

Reply:
I think it'd be good to have all this as a single PR. You can close this one and keep the other one open as is.